core/block/editor/file/file.go

```diff
@@ -162,6 +164,10 @@ func (sf *sfile) upload(s *state.State, id string, source FileSource, isSync boo
 			SetName(source.Name).
 			SetLastModifiedDate()
 	}
```

Comment:
Default style value is 0 (what is init value) and if we call this method, we enforce upload to use this style, so WithFileStyle  is way to avoid that

Reply:
What do you want to achieve?

Default 0 is Auto
```
BlockContentFile_Auto  BlockContentFileStyle = 0
```

Later we have this logic
```
if u.fileStyle == model.BlockContentFile_Auto {
		if u.fileType == model.BlockContentFile_File || u.fileType == model.BlockContentFile_None {
			u.fileStyle = model.BlockContentFile_Link
		} else {
			u.fileStyle = model.BlockContentFile_Embed
		}
	}
```